### PR TITLE
Testimonial on landing page

### PR DESCRIPTION
**Pull request by dajinchu**
We should move the testimonial content into a content file and load it in. However, because there is bold and italics, it should probably be in a markdown file instead of json. Might need to wait on #27 to add markdown support?

### CODE REVIEW

**Comment by ryandrew14** [comment]
Yeah, I have most of the styling done for the join page. I will do the graphQL stuff with MD tomorrow and then we can get this merged.

**Comment by ArunJ98** [comment]
I think the "look no further than Sandbox" should be bold too instead of italics. Also should the background be blue so we stick to the alternating backgrounds?

File: src/components/IndexPage/testimonial.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+`
+
+const AuthorInfo = styled(Body)`
+  text-align: right;
+  font-style: italic;
+`
+
+const Testimonial = ({ quote, author, role }) => (
+  <SectionPad>
+    <HeaderLineBelow>Work with us</HeaderLineBelow>
```
Comment (ryandrew14) [inline src/components/IndexPage/testimonial.js]: We might not need to have a header here. What do you think about that idea?

Comment (dajinchu) [reply src/components/IndexPage/testimonial.js]: check it out. idk if it's better

File: src/components/IndexPage/testimonial.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+const Testimonial = ({ quote, author, role }) => (
+  <SectionPad>
+    <Content>
+      <Body dangerouslySetInnerHTML={{ __html: quote }} />
+      <AuthorInfo>{author}</AuthorInfo>
+      <AuthorInfo>{role}</AuthorInfo>
+      <br />
+      <Body>
+        Think you could take advantage of computation in your research? Please
```
Comment (ryandrew14) [inline src/components/IndexPage/testimonial.js]: Think you could take advantage of computation in your research?
<Button>Work with us</Button>

Is more of a call to action and I think would get more responses. If you wanna leave it as is, there's a missing period after the second sentence.

File: src/components/IndexPage/testimonial.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+        only takes a few minutes!
+      </Body>
+      <ButtonContainer>
+        <Button href={form}>Work with us</Button>
+      </ButtonContainer>
+    </Content>
+  </SectionPad>
+)
+
+Testimonial.propTypes = {
```
Comment (ryandrew14) [inline src/components/IndexPage/testimonial.js]: Lol... I forgot to do prop types on like literally anything recently. RIP

**Pull request merged**